### DISCUSSION TITLE
Wagtail Grapples: error at url http://127.0.0.1:8000/graphql/

### DISCUSSION THREAD

**Discussion by paragliding-nsk** [torchbox/wagtail-grapple#352] [Q&A] [answered]
Hello, may I ask for help: what I'm doing wrong?

After installing wagtail-grapple and opening url http://127.0.0.1:8000/graphql/ I have only white screen and message:
{"errors":[{"message":"Must provide query string."}]}

Is it possible to change it into something like this:
![изображение](https://github.com/torchbox/wagtail-grapple/assets/59501339/49886bc1-8b0a-43d6-9b94-fa1f28110cd5)

**Comment by zerolab** [comment, marked as answer]
`/graphql` is the API endpoint. if you want the graphical interface, use `/graphiql`

**Reply by paragliding-nsk** [reply]
Thanks a lot!

**Comment by VladimirLevadnij** [comment]
I want to add that the` /graphiql` link worked for me only after I added the line `"EXPOSE_GRAPHIQL": True` to the Wagtail Grapples configuration.
